data/coalition/coalition missions.txt

```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+			`	As soon as she spots you, she quickly comes to you and says, "Human! The visitor I've heard about, you are! From the outside! A Jump Drive you have, if our space you've reached, yes?"`
+			`	You nod, and she continues, "Live on <planet>, a Saryd friend of mine does. Forgotten that his birthday was on <date>, I had! Make it in time for the party, I cannot. Busy with work, I am. At least send his present, I'd like to, but make it in time, regular ships here can't! With your Jump Drive, get there in a day you could. Help me would you, Captain?"`
+			choice
+				`	"Just that small package? Easy enough, I'll do it."`
+					accept
+				`	"I'm not headed there, sorry, have another mission to do. Somewhere else."`
+					decline
+	on complete
+		payment 36750
+		"coalition jobs" ++
+		conversation
+			`Apparently this Saryd in question doesn't live here on the spaceport village, so you must take a train to go to his home. Thankfully a train ticket was attached to the package, so you waste no time and get on the next train there.`
+			`	Even for a world without too large a population, the train wagons here feel unusually empty, with only a handful of other passengers on the one you settle in. The trip takes a few hours, so you have plenty of time to look out the window and gaze at the sights of the seemingly infinite winter forest.`
+			`	Halfway through a blizzard makes it tough to see much outside. The ticket inspectors come through again, but now serving something akin to hot chocolate, only it is instead a blue, creamy substance.`
+			`	Once you arrive at the village, you follow the makeshift map of the streets the Kimek included, and deliver the present to her Saryd friend. Apparently his birthday had just started.`
+			`	When you get back to your ship, <payment> are transferred to your account, and you get a short message from the Kimek researcher, thanking you for helping out.`
+
+
+
+mission "Saryd Students 1"
+	minor
+	name "Acquire Arachi Rum"
+	description "Some Saryd students on <origin> have offered to pay you for bringing them a shipment of traditional Arachi Rum, currently only sold on <destination>."
+	deadline
+	to offer
+		has "Coalition: First Contact: done"
+	source "Shadow of Leaves"
+	destination "Weir of Glubatub"
+	on offer
+		conversation
+			`A young Saryd bumps into you as you're passing by a nearby shop, knocking you to the ground.`
+			`	They help you up, apologizing. A couple others that were with him entered the shop without even noticing you, but almost immediately come back out.`
+			`	"No luck?" the translation machine used by the one who bumped into you translates, still active.`
+			`	His colleagues shake their heads, "Shipping here, the Arachi no longer are."`
+			`	Noticing you're still there, the Saryd decides to ask, "Human, a merchant Captain you are, correct? Looking for work, you are?"`
```

Comment:
can someone confirm the change? since Captain is always capitalized when used to refer to the player, I did so here.

Reply:
After looking into it, I don't think this one is capitalized. A good rule of thumb might be to ask if `<first> <last>` could be substituted in place of `captain`. That's just something I just came up with, so it might not be correct 100% of the time. If so, it's capitalized, and if not, it's lowercase.